azerothcore/azerothcore-wotlk
# refactor(ItemTemplate): add missing enums + other minor improvements

Partially based on: https://github.com/TrinityCore/TrinityCore/commit/1c7f0992973c3c05b0560ff6f7e1e89fcbed57f0

### How to test:

- build the core, run, login in game and have a quick look at Items ( no need to test much, just a generic check )
